shopping_mall/backend/app/services/faq_sync.py

```
@@ -0,0 +1,201 @@
+            for col_name in _ALL_COLLECTIONS:
+                try:
+                    col = client.get_collection(name=col_name)
+                    res = col.get(include=["documents"])
+                    for doc_text, doc_id in zip(
+                        res.get("documents", []), res.get("ids", [])
+                    ):
+                        corpus.append(tokenize_ko(doc_text))
+                        ids.append(doc_id)
+                        cols.append(col_name)
+                except Exception:
+                    pass  # 컬렉션 미존재 — 건너뜀
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)